[2022-07-05 07:21:39,506] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 07:21:39,531] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 07:21:39,531] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:39,531] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:39,532] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:39,568] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 03:00:00+00:00
[2022-07-05 07:21:39,580] {standard_task_runner.py:52} INFO - Started process 606 to run task
[2022-07-05 07:21:39,602] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T03:00:00+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpfz7l_26p', '--error-file', '/tmp/tmp08_txl7c']
[2022-07-05 07:21:39,603] {standard_task_runner.py:77} INFO - Job 256: Subtask kimuksung_test
[2022-07-05 07:21:39,778] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:39,963] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T03:00:00+00:00
[2022-07-05 07:21:39,964] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:39,965] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:40,000] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T030000, start_date=20220705T072139, end_date=20220705T072139
[2022-07-05 07:21:40,060] {standard_task_runner.py:92} ERROR - Failed to execute job 256 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:40,122] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:40,242] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:50,849] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 07:25:50,899] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 07:25:50,913] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:50,913] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:50,913] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:50,950] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 03:00:00+00:00
[2022-07-05 07:25:50,964] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T03:00:00+00:00', '--job-id', '287', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp02r8w9_5', '--error-file', '/tmp/tmpxcy9lxq9']
[2022-07-05 07:25:50,965] {standard_task_runner.py:77} INFO - Job 287: Subtask kimuksung_test
[2022-07-05 07:25:50,958] {standard_task_runner.py:52} INFO - Started process 796 to run task
[2022-07-05 07:25:51,233] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:51,471] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T03:00:00+00:00
[2022-07-05 07:25:51,473] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:51,474] {test.py:22} INFO - pubg
[2022-07-05 07:25:51,474] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:51,525] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T030000, start_date=20220705T072550, end_date=20220705T072551
[2022-07-05 07:25:51,626] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:51,738] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:32,887] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 08:06:32,920] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 08:06:32,921] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:32,921] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:32,921] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:32,980] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 03:00:00+00:00
[2022-07-05 08:06:33,003] {standard_task_runner.py:52} INFO - Started process 2195 to run task
[2022-07-05 08:06:33,041] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T03:00:00+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpn2629a54', '--error-file', '/tmp/tmpr19dwye_']
[2022-07-05 08:06:33,042] {standard_task_runner.py:77} INFO - Job 346: Subtask kimuksung_test
[2022-07-05 08:06:33,333] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:33,544] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T03:00:00+00:00
[2022-07-05 08:06:33,545] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:33,546] {test.py:22} INFO - pubg
[2022-07-05 08:06:33,546] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:33,646] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T030000, start_date=20220705T080632, end_date=20220705T080633
[2022-07-05 08:06:33,742] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:33,867] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:41,663] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 08:11:41,760] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [queued]>
[2022-07-05 08:11:41,760] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:41,760] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:41,760] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:41,840] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 03:00:00+00:00
[2022-07-05 08:11:41,872] {standard_task_runner.py:52} INFO - Started process 2491 to run task
[2022-07-05 08:11:41,922] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T03:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpmqipcf_h', '--error-file', '/tmp/tmpv4q97urf']
[2022-07-05 08:11:41,923] {standard_task_runner.py:77} INFO - Job 408: Subtask kimuksung_test
[2022-07-05 08:11:42,389] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T03:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:42,904] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T03:00:00+00:00
[2022-07-05 08:11:42,921] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:42,921] {test.py:22} INFO - pubg
[2022-07-05 08:11:42,922] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:43,146] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T030000, start_date=20220705T081141, end_date=20220705T081143
[2022-07-05 08:11:43,360] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:43,613] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
